### ISSUE TITLE
Some bugs when press SHIFT+CTRL+right click to place some stuff on the ground

### ISSUE THREAD

**Issue by kousakak** [anegostudios/VintageStory-Issues#2946]
### Game Version

1.18.6

### Platform

Windows

### Modded

Vanilla

### SP/MP

Singleplayer

### Description

In several versions ago press Shift+Ctrl+Right Click to place some item such as ignots,boards,peat bricks and firewoods on the bolck top will place multiple numbers of them on the block top,but now it's like that in the video I uploaded.And when I place them on the ground,the number of items in the inventory are incorrectly reduced,and the textures of placed item changes strangely when I placing items again.
Maybe I don't desturibe it correctly,so I uploaded a video.

### Screenshots

https://github.com/anegostudios/VintageStory-Issues/assets/60339803/786c2712-3245-4c2d-a137-f960fe1dc68c

### Logs

```Log```

**Comment by VallenFrostweaver** [comment]
Can confirm this is accurate.

**Comment by Barhandar** [comment]
The bug with GroundStorable where Ctrl-Shift-click placing a new pile places the `transferQuantity` amount of items on the ground, rather than `bulkTransferQuantity` as is previous behavior, is still present in 1.19. 

The biggest interference is with firing bricks, as 12 bricks can be fired, but 13 cannot, so one has to be removed every time.

**Comment by Craluminum2413** [comment]
@Barhandar Have you tested it in 1.19.4-rc.1?

**Comment by Barhandar** [comment]
@Craluminum2413 yes, the issue persists in 1.19.4-rc.1, ctrl-shift-clicking on the empty ground places 2 firewood rather than expected 8 (ignores Ctrl modifier). The changelog also does not mention it at all - "piles could be invisible when freshly placed" is unrelated.

**Comment by sekelsta** [comment]
This has been fixed at some point before 1.21.0-rc.3
